xml/smtp_banners.xml

```diff
@@ -1647,4 +1648,93 @@ The system or service fingerprint with the highest certainty overwrites the othe
     <param pos="0" name="service.product" value="Unknown"/>
     <param pos="1" name="host.name"/>
   </fingerprint>
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+        <fingerprint pattern=”^(\w+.\w+.\w)+ Service not available$>
+        <description>match to Service not available error
+        <example service.version=”4.3.2”> 4.3.2 Service not available <example>
+        <param pos="1" name="service.version"/>
+        <param pos=”0” name=”service.vendor” value=”new”/>
+        <param pos=”0” name=”service.family” value=”version”/>
+        <param pos=”0” name=”service.product” value=”version”/>
+        <fingerprint>
+         <fingerprint pattern="^(.+.{4})-\w.+echosting+[^ ]+(cafe+.+.+.[a-z]+[a-z]+[a-z])+ ESMTP$">
+        <description>match to echosting or ESMTP
+        <example service.version="e0000" website.version="cafe24.com"> e0000-e.echosting.cafe24.com ESMTP <example>
+        <param pos="1" name="service.version"/>
+        <param pos="0" name="service.vendor" value="new"/>
+        <fingerprint>
+         <fingerprint pattern="^([a-z]+[a-z]+[a-z]+.)+.cloud.netart +ESMTP +ready$">
+        <description>match to cloud.netart and ESMTP and ready state
+        <example service.version="lbc4"/> lbc4.cloud.netart ESMTP ready <example>
+        <param pos="1" name="service.version"/>
+        <param pos="0" name="service.vendor" value="new"/>
+        <param pos="0" name="service.family" value="version"/>
+        <fingerprint>
+         <fingerprint pattern="^([A-Z]+[A-Z]+[A-Z]+[A-Z])+[^ ]+(\d+[^ ]+\d) (\d\d\d) +Unauthorized$">
+        <description> match to HTTP error
+        <example service.version="HTTP" service.number="1.0" error.number="403"> HTTP/1.0 403 Unauthorized <example>
+        <param pos="1" name="service.version"/>
+        <param pos="2" name="service.number"/>
+        <param pos="3" name="error.number"/>
+        <fingerprint>
```

Comment:
The above line and similar occurrences at the end of the fingerprint should be a closing `</fingerprint>` tag.